manifest.sh

```diff
@@ -0,0 +1,2 @@
+export CLI_VERSION="0.5.19"
```

Comment:
Let's keep it as a manifest.  Env vars are effectively globals, and we should minimize them.